rpm/e2fsprogs.spec

```diff
@@ -2,15 +2,17 @@
 
 Summary: Utilities for managing ext2, ext3, and ext4 filesystems
 Name: e2fsprogs
-Version: 1.46.5
+Version: 1.47.0
 Release: 1
 # License tags based on COPYING file distinctions for various components
 License: GPLv2
 Source0: http://downloads.sourceforge.net/%{name}/%{name}-%{version}.tar.xz
 Source1: ext2_types-wrapper.h
 Patch1: 0001-Fix-incompatible-tests.-Tests-incompatible-with-curr.patch
 Patch2: 0002-Fix-build-of-tests-using-diff-from-busybox.patch
-Patch3: 0003-Revert-mke2fs.conf-enable-metadata_csum-by-default.patch
+Patch3: 0003-Revert-Fix-typo-in-defeult-etc-mke2fs.conf-file.patch
+Patch4: 0004-Revert-mke2fs.conf-enable-the-metadata_csum_seed-and.patch
+Patch5: 0005-Revert-mke2fs.conf-enable-metadata_csum-by-default.patch
```

Comment:
Three patches seems quite heavy for changing one line, especially when there's a typo and its fix included.

Could just merge into one with some explanation why there's delta? Now some rationale is getting even removed from a patch, why's that?

Reply:
I combined these 3 patches into one.